### PR TITLE
[16.0][mig] web_company_color: Migration to 16.0

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

**Comment by hbrunn** [comment]
is this a duplicate of #2404?

**Review by atchuthan — APPROVED** [review]
Functional :+1:

**Review by CasVissers-360ERP — APPROVED** [review]
Functional review

File: web_company_color/models/ir_qweb.py [hunk 1]
```diff
@@ -29,12 +29,17 @@ def _generate_asset_nodes_cache(
             res += [asset.get_company_color_asset_node()]
         return res
 
-    def _get_asset_content(
-        self, bundle, nodeAttrs=None, defer_load=False, lazy_load=False
-    ):
+    @tools.ormcache(
```
Comment (pedrobaeza) [inline web_company_color/models/ir_qweb.py:32]: Why do you need this?

**Comment by pedrobaeza** [comment]
Sorry, #2527 was merged accidentally before seeing this one (although I cancelled the action). Can you please check if there's something missing there and comment back?

**Comment by dsolanki-initos** [comment]
Hi @pedrobaeza, I've functionality tested the module PR #2527 and found that unaligned button view. For the same I've attached an image, kindly check it.
image

The same is already fixed in my PR.
image

**Comment by pedrobaeza** [comment]
Please submit the FIX in a new PR.

**Comment by dsolanki-initos** [comment]
> Please submit the FIX in a new PR.

I've created a FIX of the web_company_color https://github.com/OCA/web/pull/2530, Kindly check it.

**Pull request closed without merging**